lib/std/zig/AstGen.zig

```diff
@@ -3156,6 +3156,9 @@ fn deferStmt(
     const have_err_code = scope_tag == .defer_error and payload_token != 0;
     const sub_scope = if (!have_err_code) &defer_gen.base else blk: {
         const ident_name = try gz.astgen.identAsString(payload_token);
+        if (std.mem.eql(u8, tree.tokenSlice(payload_token), "_")) {
+            return gz.astgen.failTok(payload_token, "discard of tag capture; omit it instead", .{});
```

Comment:
tag? shouldnt it be `discard of error capture`?

Reply:
:man_facepalming: